app/app.go

```diff
@@ -612,7 +430,7 @@ func NewOsmosisApp(
 	app.SetAnteHandler(
 		NewAnteHandler(
 			app.AccountKeeper, app.BankKeeper,
-			app.TxFeesKeeper, app.GAMMKeeper,
+			*app.TxFeesKeeper, app.GAMMKeeper,
```

Comment:
?

Reply:
This is because our NewAnteHandler struct depends on the concrete struct (theres no interface for it to match)

Let me fix this to be pointer, good catch!